bin/ch-run.c

```diff
@@ -256,7 +256,9 @@ void enter_udss(char * newroot, bool writable, struct bind * binds,
    // Pivot into the new root
    TRY (0 > asprintf(&path, "%s/oldroot", newroot));
    TRY (chdir(newroot));
-   TRY (syscall(SYS_pivot_root, newroot, path));
+   // This is not the best way to handle the error, but prevents adding dir checking code -JLW
+   TRX (syscall(SYS_pivot_root, newroot, path),
+         "Maybe /oldroot doesn't exist?");
```

Comment:
`pivot_root(2)` has such an huge list of failure modes that I think a hint could prove confusing. My guess is that a few lines to `stat(2)` or `opendir(2)`/`closedir(2)` would be worthwhile?